assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -21,20 +21,20 @@
 	@include googlesitekit-inner-padding;
 	@include shadow;
 	background-color: $c-base;
+	left: 0;
+	position: sticky;
+	right: 0;
+	top: 0;
 	z-index: 12;
 
-	// only become sticky when WordPress's #wpadminbar becomes sticky
-	@media (min-width: $bp-tablet) {
-		left: 0;
-		position: sticky;
-		right: 0;
-		top: 46px;
-	}
-
 	@media (min-width: $bp-wpAdminBarTablet) {
 		top: 32px;
 	}
 
+	@media (min-width: $bp-tablet) {
+		top: 46px;
+	}
```

Comment:
This is the correct rule but should come before the previous media query as it is currently resulting in a gap in larger viewports.
![image](https://user-images.githubusercontent.com/1621608/100591274-2f34bf00-32fe-11eb-9eeb-d89b687f032f.png)

![image](https://user-images.githubusercontent.com/1621608/100591280-3065ec00-32fe-11eb-9886-9d67e8a0ab65.png)

Reply:
Good spot.